packages/editor-ui/src/components/Sticky.vue

```diff
@@ -75,17 +75,18 @@
 						<div
 							class="color"
 							data-test-id="color"
-							v-for="(color, index) in availableColors"
+							v-for="(_, index) in Array.from({ length: 7 })"
 							:key="index"
-							v-on:click="changeColor(color)"
+							v-on:click="changeColor(index + 1)"
+							:class="`sticky-color-${index + 1}`"
 							:style="{
-								'background-color': color,
 								'border-width': '1px',
 								'border-style': 'solid',
 								'border-color': 'var(--color-text-dark)',
 								'box-shadow':
-									(index === 0 && node?.parameters.color === '') || color === node?.parameters.color
-										? `0 0 0 1px ${color}`
+									(index === 0 && node?.parameters.color === '') ||
+									index + 1 === node?.parameters.color
+										? `0 0 0 1px var(--sticky-color-${index + 1})`
```

Comment:
ah nice here.. maybe we can add the background-color directly here too.. same way...